zambeze/orchestration/message/message_factory.py

```diff
@@ -1,50 +1,34 @@
-# Local imports
-from .abstract_message import AbstractMessage
-from .activity_message.message_activity import MessageActivity
-from .activity_message.message_activity_validator import MessageActivityValidator
-from .activity_message.message_activity_template_generator import createActivityTemplate
-from .status_message.message_status import MessageStatus
-from .status_message.message_status_validator import MessageStatusValidator
-from .status_message.message_status_template_generator import createStatusTemplate
-from zambeze.orchestration.plugins_message_validator import PluginsMessageValidator
-from zambeze.orchestration.plugins_message_template_engine import (
-    PluginsMessageTemplateEngine,
-)
-
-from ..zambeze_types import MessageType, ActivityType
-
-# Standard imports
 import logging
-import uuid
 from typing import Optional
 
+from .message_activity import MessageActivity
+from .message_status import MessageStatus
+from .message_activity_validator import MessageActivityValidator, createActivityTemplate
+from .message_status_validator import MessageStatusValidator, createStatusTemplate
+from .abstract_message import AbstractMessage
+from ..zambeze_types import MessageType
+
 
 class MessageFactory:
-    def __init__(self, logger: Optional[logging.Logger] = None):
-
+    def __init__(self, plugins, logger: Optional[logging.Logger] = None):
         self._logger = logger
-        self._plugins_message_template_generators = PluginsMessageTemplateEngine(logger)
-        self._plugins_message_validators = PluginsMessageValidator(logger)
+        self._plugins = plugins
```

Comment:
This change is incorrect, the template generator and message validator were separated from the plugins object intentionally. The MessageFactory only needs to be able to generate templates and validate them, it doesn't need to run the plugins which was the reason for the change. Hopefully, that makes sense.

Reply:
Though I don't know if the gitdiff is working correctly here or not still, so this is just a guess. Hopefully, our patch command isn't messing with us here.